# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/shade.rs
# Commit: 0c03e3ad5e56
# Full Hash: 0c03e3ad5e56201e35437734b0ae1d34faa52005
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-12-15 09:29:54
# Regressor Bug: 1682365
# File Overlap Count: 1
# Description:
#   Bug 1682365 - Pt 1 - Remove PrevPassAlpha and PrevPassColor r=nical
#   
#   Remove usage of the implicit prev pass alpha and color texture
#   samplers from batching / renderer / shader code. They are replaced
#   by explicit references to the texture ID for the source task.
# ==============================================================================

diff -r 4e0ba157c016 -r 0c03e3ad5e56 gfx/wr/webrender/src/shade.rs
--- a/gfx/wr/webrender/src/shade.rs	Sat Dec 12 22:13:43 2020 +0000
+++ b/gfx/wr/webrender/src/shade.rs	Mon Dec 14 19:38:48 2020 +0000
@@ -258,8 +258,6 @@
                             ("sColor1", TextureSampler::Color1),
                             ("sColor2", TextureSampler::Color2),
                             ("sDither", TextureSampler::Dither),
-                            ("sPrevPassAlpha", TextureSampler::PrevPassAlpha),
-                            ("sPrevPassColor", TextureSampler::PrevPassColor),
                             ("sTransformPalette", TextureSampler::TransformPalette),
                             ("sRenderTasks", TextureSampler::RenderTasks),
                             ("sGpuCache", TextureSampler::GpuCache),
